### PR TITLE
Unit tests review and Connect Back for Settings 

### PR DESCRIPTION
**Pull request by avolchivschi**

### CODE REVIEW

File: intellst-front/src/store/login.js [hunk 1]
```diff
@@ -62,15 +62,31 @@ const login = {
...
-          commit("setUser", { ...data[0] });
+          commit("setUser", { ...data });
         })
         .catch(() => {
           // localStorage.removeItem("token");
           // router.push("/login");
         });
     },
+    setSettings({ commit }, payload) {
+      Axios.post("/api/entreprises/1", {
```
Comment (andreikojusko) [inline intellst-front/src/store/login.js]: why enterprise ID is hardcoded?

File: intellst-front/src/views/Settings.vue [hunk 2]
```diff
@@ -93,12 +94,19 @@ export default {
...
+      temperature: "",
+      restrictionPeriod: "",
+    },
   }),
   methods: {
+    ...mapActions("login", ["setSettings"]),
     submit() {
       this.$refs.observer.validate();
+      this.setSettings(this.entreprise);
+      setTimeout(() => {
```
Comment (andreikojusko) [inline intellst-front/src/views/Settings.vue]: we need to discuss this beviour

**Review by andreikojusko — CHANGES_REQUESTED** [review]

File: intellst-front/src/store/login.js (hunk 1 above)
Comment (avolchivschi) [reply intellst-front/src/store/login.js]: Modified

File: intellst-front/src/views/Settings.vue (hunk 2 above)
Comment (avolchivschi) [reply intellst-front/src/views/Settings.vue]: Could you check it please, it works now

**Pull request merged**